daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ValueEnricher.scala

```diff
@@ -65,14 +65,10 @@ final class ValueEnricher(
   ): Result[Value] = for {
     // TODO: https://github.com/digital-asset/daml/issues/14112
```

Comment:
You can remove this todo and maybe even close the issue if the other todos & stuff in there are also addressed by this.

Reply:
This todo is done, but some require small tweaks before I can get rid of them
Making another PR for the small fixes necessary to eliminate the remaining todos